int32validator/any_with_all_warnings_example_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package int32validator_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/int32validator"
+)
+
+func ExampleAnyWithAllWarnings() {
+	// Used within a Schema method of a DataSource, Provider, or Resource
+	_ = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"example_attr": schema.Int32Attribute{
+				Required: true,
+				Validators: []validator.Int32{
+					// Validate this Int32 value must either be:
+					//  - 1.0
+					//  - At least 2.0
+					int32validator.AnyWithAllWarnings(
+						int32validator.OneOf(1.0),
+						int32validator.AtLeast(2.0),
+					),
```

Comment:
nit: Same comment as above (using integer constants instead of float)
```suggestion
					// Validate this Int32 value must either be:
					//  - 1
					//  - At least 2
					int32validator.AnyWithAllWarnings(
						int32validator.OneOf(1),
						int32validator.AtLeast(2),
					),
```